lib/repl.js

```diff
@@ -1119,6 +1176,17 @@ function defineDefaultCommands(repl) {
       this.displayPrompt();
     }
   });
+
+  repl.defineCommand('editor', {
+    help: 'Entering editor mode (^D to finish, ^C to cancel)',
+    action() {
+      if (!this.terminal) return;
+      this.editorMode = true;
+      REPLServer.super_.prototype.setPrompt.call(this, '');
+      this.outputStream.write(
+        '// Entering editor mode (^D to finish, ^C to cancel)\n');
```

Comment:
@thefourtheye It helps user to remember ^D for finish/save and  ^C for cancel the input!